weave

```diff
@@ -1795,18 +1814,40 @@ case "$COMMAND" in
     # intentionally undocumented since it assumes knowledge of weave
     # internals
     create-bridge)
+        cat 1>&2 <<EOF
```

Comment:
Hmm. So this PR doesn't actually make it impossible to call `create-bridge` when docker isn't running. So I suggest we do issue a deprecation warning instead:

"WARNING: `weave create-bridge` is deprecated. Instead use `weave attach-bridge` to link the docker and weave bridges after both docker and weave have been started. Note that, unlike `weave create-bridge`, `weave attach-bridge is compatible with fast data path."

(modulo formatting)